package.json

```diff
@@ -40,7 +40,9 @@
         "test:e2e:docker": "npm-run-all --serial test:e2e:docker:build \"test:e2e:docker:run {@}\" --",
         "test:electron": "jest --projects src/tests/electron --runInBand",
         "watch": "npm-run-all --parallel --race --print-label watch:scss watch:grunt watch:test watch:webpack-dev-browser watch:webpack-electron",
-        "watch:build": "npm-run-all --parallel --race --print-label watch:scss watch:grunt watch:webpack-dev-browser watch:webpack-electron",
+        "watch:build:all": "npm-run-all --parallel --race --print-label watch:scss watch:grunt watch:webpack-dev-browser watch:webpack-electron",
+        "watch:build:web": "npm-run-all --parallel --race --print-label watch:scss watch:grunt watch:webpack-dev-browser",
+        "watch:build:android": "npm-run-all --parallel --race --print-label watch:scss watch:grunt watch:webpack-electron",
```

Comment:
I think `watch:build:electron` would be more consistent with the other run scripts

Reply:
udpated